Q1/imyogeshgaur/run.md

```diff
@@ -0,0 +1,17 @@
+To run C++ program you have to install g++ compiler after installing it type the following command in your Terminal.
+
+<br/>
```

Comment:
Not required here. For more information regarding markdown format, check out [markdown-cheetsheet](https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md)